.travis.yml

```diff
@@ -34,42 +34,44 @@ matrix:
   include:
     # Linux {
 
-    - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=clang-cxx17
+    # This version of Chaiscript (6.0) is incompatible with C++17
+    #- os: linux
+    #  env: PROJECT_DIR=examples/chaiscript TOOLCHAIN=clang-cxx17
 
     - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=gcc-7-cxx17
+      env: PROJECT_DIR=examples/chaiscript TOOLCHAIN=gcc-7-cxx14
```

Comment:
Please keep old toolchains name. If toolchain is not working comment it out. Add any useful toolchains after main table and guard them with `# Extra {` comment. This will help with the merges in future. See example:
* https://github.com/ingenue/hunter/blob/df35840353ca252bd4d13b488ddc44c4df5e1a02/.travis.yml#L80-L88

Reply:
Will update